lib/python/qmk/keyboard.py

```diff
@@ -169,6 +169,7 @@ def render_layout(layout_data, render_ascii, key_labels=None):
         y = ceil(key.get('y', 0) * 3)
         w = ceil(key.get('w', 1) * 4)
         h = ceil(key.get('h', 1) * 3)
+        is_iso_key = w == 5 and h == 6
```

Comment:
Less dependent on the previous multiplication?
```suggestion
        is_iso_key = key.get('w', 1) == 1.25 and key.get('h', 1) == 2
```

However the above code is already a bit magic

Reply:
That's what I originally had, but now we're doing the `.get()` twice. I figured the variable name was indication enough.

Ideally some of this code should be refactored out into "render a key" and possibly "render the ISO enter key" functions.

Reply:
I mention, as #15846 modifies this area and it wouldnt be addressed in a simple rebase.